app/main.py

```diff
@@ -4,4 +1,37 @@
+import os
+import requests
+
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
+def get_weather(api_key: str) -> None:
+    print(f"Performing request to Weather API for city {CITY}...")
+    params = {
+        "key": api_key,
+        "q": CITY
+    }
+    response = requests.get(API_URL, params=params)
+    if response.status_code == 200:
+        data = response.json()
+        location_time = data["location"]["localtime"]
+        temperature = data["current"]["temp_c"]
+        condition_text = data["current"]["condition"]["text"]
+        location = data["location"]["name"]
+        country = data["location"]["country"]
+        print(
+            f"{location}/{country} {location_time} Weather: "
+            f"{temperature} Celsius, "
+            f"{condition_text}"
+        )
+    else:
+        print("Error:", response.status_code, response.text)
```

Comment:
Code Quality Issue: You should consider implementing error handling for the requests.get() call. This function can raise a requests.exceptions.RequestException if a network problem occurs (like a DNS resolution failure, refused connection, etc). You could use a try/except block to catch this exception and print an error message.